### ISSUE TITLE
Windows - ERR_UNSUPPORTED_ESM_URL_SCHEME

### ISSUE THREAD

**Issue by Pecral** [storybook-vue/storybook-nuxt#71]
Problems

- Checkout https://github.com/storybook-vue/storybook-nuxt-starter master
- update "@storybook-vue/nuxt" to 0.1.5
- pnpm install
- pnpm storybook

![image](https://github.com/storybook-vue/storybook-nuxt/assets/16672910/d9380322-60dd-4ecc-8fe6-fcd2671cc184)

Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'

Environment

- On windows 10/11
- Nodejs 18.17.1 or Nodejs 20.6.1

The problem was reproducible on two different windows 10/11 machines.

Do you have any ideas?

**Comment by Ingramz** [comment]
I'm encountering this issue as well. Quick debugging led me to the following line https://github.com/storybook-vue/storybook-nuxt/blob/d6ddde06ec778a7128e484bf668975aef8b4d558/packages/storybook-nuxt/src/preset.ts#L132 where `require.resolve` returns full path to file, generally starting with `C:` on windows, which the import function does not like. Other usages of `import(require.resolve(..))` fail in similar manner.

Is there any reason why `await import('@storybook/vue3-vite/preset')` can't just be used?

**Comment by chakAs3** [comment]
@tobiasdiez i will do some tests on Windows, would please create a reproduction repo for me?

**Comment by tobiasdiez** [comment]
Perfect, I think it should already occur in the demo (see issue description), but here is also a real world repo: https://github.com/JabRef/JabRefOnline/pull/2244

**Comment by chakAs3** [comment]
> where `require.resolve` returns full path to file, generally starting with `C:` on windows, which the import function does not like. Other usages of `import(require.resolve(..))` fail in similar manner.

you are completely right, this was some legacy code from storybook codebase, i did not pay attention to it since it is not breaking on Mac,  but yeah should work fine, Thanks @Ingramz

**Comment by Ingramz** [comment]
`0.1.6-alpha.5` didn't work, but `0.1.6-alpha.6` seems to be fine.

`build-storybook` finishes without errors and `storybook` starts up the development web server, but there's something else wrong that doesn't build or load the stories properly. But that's out of the scope of this specific issue, which seems to be solved now.

**Comment by daviddomkar** [comment]
Hello. I am still encountering this issue with latest version on Windows.

**Comment by daviddomkar** [comment]
From additional testing I can confirm that version 0.2.1 works. But 0.2.2 and newer throws this exact error.
